Makefile

```diff
@@ -10,7 +10,7 @@ init:
 	cd dleq/cgo-dleq && make build
 
 .PHONY: lint
-lint: init
```

Comment:
We'll just assume people run `make init` before running `make lint`.  I run the command a lot and it was too noisy and time consuming to have those other scripts run.